misc/RELEASE_CHECKLIST_TEMPLATE.md

```diff
@@ -0,0 +1,83 @@
+
+- [ ] Update: `git fetch && git checkout develop && git pull`
+- [ ] Rebase the `master` branch on the `develop` branch: `git checkout master && git rebase develop`
+- [ ] Generate new dist files: `npm prune && rm -rf node_modules && npm install && npm run build && npm run test && git commit -am "generated dist files for vX.X.Y"
+- [ ] Create a version tag: `git tag "vX.X.Y"`
+- [ ] [Remove the protection](https://github.com/almende/vis/settings/branches/master) from `master`.
+- [ ] FORCE-Push the branches to github: `git push --force && git push --tag`
+- [ ] [Re-Enable branch protection](https://github.com/almende/vis/settings/branches/master) (enable ALL checkboxes) for `master`.
+- [ ] Publish with npm: `npm publish` (check [npmjs.com](https://www.npmjs.com/package/vis))
+- [ ] Create a [new Release](https://github.com/almende/vis/releases/new) with the tang and the name "vX.X.Y" and copy the data vom [HISTORY.md](../HISTORY.md) into the body.
+
+
+## Test
+- [ ] Go to a temp directory (e.g. "vis_vX.X.Y"): `cd .. && mkdir vis_vX.X.Y && cd vis_vX.X.Y`
+- [ ] Install the library from npm: `npm init -f && npm install vis`
+- [ ] Verify if it installs the just released version, and verify if it works: `cd node_modules/vis/
+- [ ] Install the library via bower: `cd ../.. && bower install vis`
+- [ ] Verify if it installs the just released version, and verify if it works: `cd bower_components/vis/`
+- [ ] Clone the master from github: `cd ../.. && git clone git@github.com:almende/vis.git`.
+- [ ] Verify if it installs the just released version, and verify if it works. `cd vis`
+
+## Update website
+- [ ] update the gh-pages branch: `git checkout gh-pages && git pull && git checkout -b "gh-pages_vX.X.Y"`
+- [ ] Copy the `dist` folder from the `master` branch to the `github-pages` branch in another directory, overwriting existing files: `cp -rf ../vis_vX.X.Y/vis/dist .`
+- [ ] Copy the `docs` folder from the `master` branch to the `github-pages` branch in another directory, overwriting existing files: `cp -rf ../vis_vX.X.Y/vis/docs .`
+- [ ] Copy the `examples` folder from the `master` branch to the `github-pages` branch in another directory, overwriting existing files: `cp -rf ../vis_vX.X.Y/vis/examples .`
+- [ ] Check if there are new or updated examples, and update the gallery screenshots accordingly.
+- [ ] Update the library version number in the `index.html` page.
+- [ ] Update the CDN links at the download section of index.html AND the CDN link at the top. (search-replace all!!)
+- [ ] Commit the changes: `git add -A && git commit -m "updates for vX.X.Y"`
+- [ ] Push the changes `git push --set-upstream origin gh-pages_vX.X.Y`
+
+## Prepare next version
+- [ ] Switch to the "develop" branch: `git checkout develop`.
+- [ ] Change version numbers in "package.json" to a snapshot version `X.X.Z-SNAPSHOT`.
+- [ ] Commit and push: `git commit -am "changed version to vX.X.Z-SNAPSHOT"`
+- [ ] Create new tag: `git tag vX.X.Z-SNAPSHOT`.
```

Comment:
@yotamberk Approved the changes. I meant that the version after vX.Y.Z should be vX.Y.Z+1-SNAPSHOT,